src/rSW_Site.c

```diff
@@ -622,28 +636,38 @@ void onSet_SW_SIT(SEXP SW_SIT, LOG_INFO* LogInfo) {
 	strcpy(v->site_ptf_name, CHAR(STRING_ELT(swrc_flags, 1)));
 	v->site_ptf_type = encode_str2ptf(v->site_ptf_name);
 	PROTECT(has_swrcp = GET_SLOT(SW_SIT, install("has_swrcp")));
-	v->site_has_swrcp = LOGICAL(has_swrcp)[0];
+	v->inputsProvideSWRCp = LOGICAL(has_swrcp)[0];

```

Comment:
onSet_SW_SIT() -- see longer comment for onGet_SW_SWRCp()
- copy new slot "depthSapric" to `v->depthSapric`